tests/e2e/forms/repeat-form.wdio-spec.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+
+const constants = require('../../constants');
+const utils = require('../../utils');
+const auth = require('../../auth')();
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+
+const xml = fs.readFileSync(`${__dirname}/../../../demo-forms/repeat-translation.xml`, 'utf8');
+const formDocument = {
+  _id: 'form:repeat-translation',
+  internalId: 'repeat-translation',
+  title: 'Repeat',
+  type: 'form',
+  _attachments: {
+    xml: {
+      content_type: 'application/octet-stream',
+      data: Buffer.from(xml).toString('base64')
+    }
+  }
+};
+const userContactDoc = {
+  _id: constants.USER_CONTACT_ID,
+  name: 'Jack',
+  date_of_birth: '',
+  phone: '+64274444444',
+  alternate_phone: '',
+  notes: '',
+  type: 'person',
+  reported_date: 1478469976421,
+  parent: {
+    _id: 'some_parent'
+  }
+};
+
+describe('RepeatForm', () => {
+  before(async () => {
+    await utils.seedTestData(userContactDoc, [formDocument]);
+  });
+
```

Comment:
You need to add an `after` here, where you resetDb. We reuse the same database across all tests, so it's important that each test suite cleans up after itself.